test/test_EKF_utils.cpp

```diff
@@ -55,3 +55,16 @@ TEST(eclPowfTest, compareToStandardImplementation)
 		}
 	}
 }
+
+TEST(euler312YawTest, fromQuaternion)
+{
+	matrix::Quatf q1(3.5f,2.4f,-0.5f,-3.f);
+	q1.normalize();
+	const matrix::Eulerf euler1(q1);
+	EXPECT_FLOAT_EQ(euler1(2), getEuler321Yaw(q1));
+
+	matrix::Quatf q2(0.f,0,-1.f,0.f);
+	q2.normalize();
+	const matrix::Eulerf euler2(q2);
+	EXPECT_FLOAT_EQ(euler2(2), getEuler321Yaw(q2));
+}
```

Comment:
Nice, can we also have a test for `shouldUse321RotationSequence` that shows that it returns false for a pitch90 condition?